paintBrush.js

```diff
@@ -0,0 +1,42 @@
+// Select the canvas in the dom
+const canvas = document.querySelector('canvas')
+// Sets the canvas to 2D drawing
+const ctx = canvas.getContext('2d')
+
+let painting = false;
+
+if (canvas) {
+    canvas.addEventListener('mousemove', drawLine);
+    // Start painting on click event
+    canvas.addEventListener('mousedown', startPainting);
+    // Stop painting on stopping clicking/off page
+    canvas.addEventListener('mouseup', stopPainting);
+    canvas.addEventListener('mouseleave', stopPainting);
+}
+
+function drawLine(event){
+    ctx.lineTo(event.offsetX, event.offsetY)
+    if (!painting) {
+        // When not painting, begin a new path
+        ctx.beginPath();
+        ctx.moveTo(event.offsetX, event.offsetY);
+    } else {
+        // When painting, draw a line to...
+        ctx.lineTo(event.offsetX, event.offsetY)
+        ctx.stroke()
+    }
+}
+
+function stopPainting() {
+    painting = false;
+}
+
+function startPainting() {
+    painting = true;
+    if (painting) {
```

Comment:
whts the point of this if statement? its always going to be true, you have just set it to true on the line above